### DISCUSSION TITLE
Any plans for a version 1.0?

### DISCUSSION THREAD

**Discussion by tomchor** [pydata/xarray#4678] [General] [answered]
Sorry if this isn't the proper place to ask this, but I've been wondering for a while about version 1.0.

I've been using the package for a while and (from my perspective at least) it looks mature and stable enough that it merits a version 1.0 (and that has been case for a while, imho). It's stable enough that I've seen several discussions on the Julia community about implementing something similar in Julia, regarding xarray as an end goal for them. I know version numbers are in the end meaningless, but I think a "whole number version" helps inspire some confidence in the community. I know I'd like to see that.

So I was wondering what's everyone's thoughts about this, especially the main developers.

**Comment by keewis** [comment, marked as answer]
I don't think there will be a 1.0 very soon. The reason for this is that there are a few big changes we want to make before that (see the [roadmap](http://xarray.pydata.org/en/stable/roadmap.html)), among them the backend refactor we're currently working on, but also the index refactor (see #1603) and the documentation overhaul / reorganization.

**Reply by shoyer** [reply]
This is our current thinking. The index refactor is the biggest change we are waiting on, because it will alter xarray's underlying data model in some subtle ways.

Another option would be to declare "1.0" now, and make xarray 2.0 after the index refactor.
